wpilib/wpilib/sensorbase.py

```diff
@@ -1,18 +1,19 @@
-# validated: 2016-12-31 JW dc9a9e5d960d edu/wpi/first/wpilibj/SensorBase.java
-#----------------------------------------------------------------------------
-# Copyright (c) FIRST 2008-2016. All Rights Reserved.
+# validated: 2017-11-23 TW e1195e8b9dab edu/wpi/first/wpilibj/SensorBase.java
+# ----------------------------------------------------------------------------
+# Copyright (c) FIRST 2008-2017. All Rights Reserved.
 # Open Source Software - may be modified and shared by FRC teams. The code
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
-#----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
 
 import hal
 
 from .livewindowsendable import LiveWindowSendable
 
 __all__ = ["SensorBase"]
 
-class SensorBase(LiveWindowSendable):
+
+class SensorBase:
```

Comment:
If you're going to remove this, you need to make everything that should be a LiveWindowSendable inherit from LiveWindowSendable…

Reply:
I think there are a lot more sensors in WPILib that you haven't touched here that should be LiveWindowSendable.

Reply:
Yeah... we should keep this I think? And document that it's python-specific.

Reply:
Is that the final verdict?

Reply:
A bunch of sensors depend on it to set up their value listeners, and this behavior isn't covered by unit tests for most of them. In the interest of getting things done, it should be kept, though personally, I would rather refactor it. As is, it isn't easy to see when this is happening, and it only works for "Value" entries; if it were an explicit call it could be more generic